site/content/en/docs/Guides/player-tracking.md

```diff
@@ -0,0 +1,112 @@
+
+The current player capacity is represented in `GameServer.Status.Players.Capacity` resource value.
+
+We can see this in action, when we look at the Status section of a GameServer resource
+, wherein the capacity has been set to 20:
+
+```
+...
+Status:
+  Address:    14.81.195.72
+  Node Name:  gke-test-cluster-default-6cd0ba67-1mps
+  Players:
+    Capacity:  20
+    Count:     0
+    Ids:       <nil>
+  Ports:
+    Name:          gameport
+    Port:          7983
+  Reserved Until:  <nil>
+  State:           Ready
+```
+
+From the SDK, the game server binary can also retrieve the current player capacity 
+via [`SDK.Alpha().GetPlayerCapacity()`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-getplayercapacity" >}}).
+
+## Connecting and Disconnecting Players
+
+As players connect and disconnect from your game, the Player Tracking functions enable you to track which players 
+are currently connected.
+
+It assumed that each player that connects has a unique token that identifies them as a player.
+
+When a player connects to the game server binary, 
+calling [`SDK.Alpha().PlayerConnect(playerID)`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-playerconnect-playerid" >}})
+with the unique player token will register them as connected, and store their player id.
+ 
+At disconnection time,
+call [`SDK.Alpha().PlayerDisconnect(playerID)`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-playerdisconnect-playerid" >}})
+, which will deregister them and remove their player id from the list.
+
+Each of these `playerIDs` is stored on `GameServer.Status.Players.IDs`, and the current count of connected players
+can be seen in `GameServer.Status.Players.Count`. 
+
+You can see this in action below in the `GameServer` Status section, where there are 4 players connected:
+
+```
+...
+Status:
+  Address:    39.82.196.74
+  Node Name:  gke-test-cluster-default-6cd0ba77-1mps
+  Players:
+    Capacity:  10
+    Count:     4
+    Ids:
+      xy8a
+      m0ux
+      71nj
+      lpq5
+  Ports:
+    Name:          gameport
+    Port:          7166
+  Reserved Until:  <nil>
+  State:           Ready
+```
+
+## Checking Player Data
+
+Not only is the connected player data stored on the `GameServer` resource, it is also stored in memory within the
+SDK, so that it can be used from within the game server binary as a realtime, thread safe, registry of connected
+players.
+
+Therefore, if you want to:
+
+* Get the current player count, call [`SDK.Alpha().GetPlayerCount()`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-getplayercount" >}})
+* Check if a specific player is connected, call [`SDK.Alpha().IsPlayerConnected(playerID)`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-isplayerconnected-playerid" >}})
+* Retrieve the full list of connected players, call [`SDK.Alpha().GetConnectedPlayers()`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-getconnectedplayers" >}})
+
+## Next Step
```

Comment:
Even though there is only one link, I think this should be plural, like on https://0ec2b41-dot-preview-dot-agones-images.appspot.com/site/docs/installation/creating-cluster/gke/